test/Feature/RpcSpec.hs

```diff
@@ -689,7 +780,7 @@ spec actualPgVersion =
 
     context "only for POST rpc" $ do
       it "gives a parse filter error if GET style proc args are specified" $
-        post "/rpc/sayhello?name=John" [json|{}|] `shouldRespondWith` 400
+        post "/rpc/sayhello?name=John" [json|{name: "John"}|] `shouldRespondWith` 400
```

Comment:
Hm, why did this test change?

Reply:
After the change, postgrest looks for the function existence first. It doesn't find a sayhello function without parameters, it gives a 404 error instead of the 400. Only when the function exists it checks for parse filter errors, that's why I added the parameter in the json payload.

Reply:
Got it. The change is harmless, so it's cool to leave it like this.